xbmc/cores/IPlayer.h

```diff
@@ -226,7 +226,9 @@ enum ViewMode {
   ViewModeStretch16x9,
   ViewModeOriginal,
   ViewModeCustom,
-  ViewModeStretch16x9Nonlin
+  ViewModeStretch16x9Nonlin,
+  ViewModeZoom120Width,
```

Comment:
Just a small suggestion, but I thiink it would be more natural to have the 110 first and then the 120